docker-compose.dev.yml

```diff
@@ -1,17 +1,21 @@
-version: '2'
+version: '2.1'
```

Comment:
2.1 isn't supported by the docker on Circle, I was thinking of moving to 2.1 locally to add healthchecks but I'll probably revert that here and try it in a different PR